lnrpc/lightning.proto

```diff
@@ -2636,6 +2636,9 @@ message PendingHTLC {
 }
 
 message PendingChannelsRequest {
+    // Indicates whether to include the raw transaction hex for
+    // waiting_close_channels.
+    bool include_tx = 1;
```

Comment:
let's be more explicit and name it `include_tx_hex` or `include_raw_tx`, seems easier to be used.

Reply:
Renamed to include_raw_tx